app/src/main/res/layout/activity_birthday.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It would be interesting to see a screenshot of this activity. Could you attach one to the PR? 🙂

Reply:
![image](https://user-images.githubusercontent.com/26031482/72726858-c206b800-3b89-11ea-9220-9db46de92594.png)
With [this](https://lottiefiles.com/13982-testing) animation

Reply:
Neat!

Reply:
Neat!